README.md

```diff
@@ -128,9 +128,9 @@ Top-level `await` lets us rely on the module system itself to handle all of thes
 ```mjs
 // awaiting.mjs
 import { process } from "./some-module.mjs";
-const dynamic = await import(computedModuleSpecifier);
-const data = await fetch(url);
-export const output = process(dynamic.default, data);
+const dynamic = import(computedModuleSpecifier);
+const data = fetch(url);
+export const output = process(await dynamic.default, await data);
```

Comment:
Doesn't it require parentheses? Like:

```javascript
process((await dynamic).default, await data);
```

Reply:
It does! Want to send a follow-up PR, @trotyl?

Reply:
@mathiasbynens Created https://github.com/tc39/proposal-top-level-await/pull/70